src/framework/testflow/internal/api/testflowapi.cpp

```
@@ -145,46 +145,46 @@ void AutobotApi::saveProject(const QString& name)
 
-void AutobotApi::async(const QJSValue& func, const QJSValueList& args)
+void TestflowApi::async(const QJSValue& func, const QJSValueList& args)
 {
     async::Async::call(this, [func, args]() {
         QJSValue mut_func = func;
         mut_func.call(args);
     });
 }
 
-int AutobotApi::randomInt(int min, int max) const
+int TestflowApi::randomInt(int min, int max) const
```

Cppcheck (2.20.0)

[style] The function 'randomInt' is never used.

(unusedFunction)